app/controllers/prisoners_controller.rb

```diff
@@ -114,22 +121,6 @@ def load_summary(summary_type)
     @offenders = sort_and_paginate(items, default_sort: :last_name)
   end
 
-  def get_slice_for_page(offender_list, user_id)
-    offenders = []
-    user_allocations = []
-
-    offender_list.map do |offender|
-      allocation = @prison.allocation_for(offender)
```

Comment:
`@prison.allocation_for(offender)` Loads a `Prison::AllocationSummary` which is populated with `Prison#policy_offenders` - 1 call to prisoner search here